### PR TITLE
Refactor backend docker-compose files, reduce number of files

### PR DESCRIPTION
**Pull request by MaciejTe**
Refactor backend docker-compose files, reduce number of files

Changelog: None

Signed-off-by: Maciej Tomczuk <maciej.tomczuk@northern.tech>

### CODE REVIEW

File: config/traefik/traefik.enterprise.yaml [hunk 1]
```diff
@@ -4,37 +4,104 @@ http:
...
       service: tenantadm
       tls: true
 
+    #
+    # auditlogs
+    #
+    auditlogs:
+      entrypoints: https
+      middlewares: #{{- if not (env "TESTING") }}
+        - circuit-breaker #{{- end}}
```
Comment (alfrunes) [inline config/traefik/traefik.enterprise.yaml:38]: (Nitpick) can be expressed more compactly using the above template block:
```suggestion
      middlewares: #{{- template "circuit-breaker"}}
```

File: config/traefik/traefik.enterprise.yaml [hunk 2]
```diff
@@ -4,37 +4,104 @@ http:
...
+      service: auditlogs
+      tls: true
+
+    #
+    # devicemonitor
+    #
+    devicemonitor:
+      entrypoints: https
+      middlewares:
+        - circuit-breaker
```
Comment (alfrunes) [inline config/traefik/traefik.enterprise.yaml:54]: I believe the `circuit-breaker` makes the full integration tests misbehave, which is why it's conditionally included for the other services.
```suggestion
      middlewares: #{{- template "circuit-breaker"}}
```

File: config/traefik/traefik.yaml [hunk 3]
```diff
@@ -184,61 +183,132 @@ http:
...
       service: useradm
       tls: true
 
+    #
+    # deviceconnect
+    #
+    deviceconnect:
+      entrypoints: https
+      middlewares:
+        - circuit-breaker
```
Comment (alfrunes) [inline config/traefik/traefik.yaml:211]:
```suggestion
      middlewares: #{{- template "circuit-breaker"}}
```

File: config/traefik/traefik.yaml [hunk 4]
```diff
@@ -184,61 +183,132 @@ http:
...
+      service: deviceconnect
+      tls: true
+
+    #
+    # deviceconfig
+    #
+    deviceconfig:
+      entrypoints: https
+      middlewares:
+        - circuit-breaker
```
Comment (alfrunes) [inline config/traefik/traefik.yaml:243]:
```suggestion
      middlewares: #{{- template "circuit-breaker"}}
```

File: config/traefik/traefik.enterprise.yaml (hunk 1 above)
Comment (MaciejTe) [reply config/traefik/traefik.enterprise.yaml]: Did that, thanks!

File: config/traefik/traefik.enterprise.yaml (hunk 2 above)
Comment (MaciejTe) [reply config/traefik/traefik.enterprise.yaml]: done, thanks!

File: config/traefik/traefik.yaml (hunk 3 above)
Comment (MaciejTe) [reply config/traefik/traefik.yaml]: done, thanks!

File: config/traefik/traefik.yaml (hunk 4 above)
Comment (MaciejTe) [reply config/traefik/traefik.yaml]: done, thanks!

**Comment by MaciejTe** [comment]
@mender-test-bot start pipeline

File: extra/release_tool.py [hunk 5]
```diff
@@ -452,10 +452,6 @@ def filter_docker_compose_files_list(list, version):
     _DOCKER_ONLY_YML = [
         "docker-compose.yml",
         "docker-compose.enterprise.yml",
-        "docker-compose.auditlogs.yml",
-        "docker-compose.connect.yml",
-        "docker-compose.config.yml",
-        "docker-compose.monitor.yml",
```
Comment (lluiscampos) [inline extra/release_tool.py:458]: As pointed out privately in Slack, these lines cannot be removed. This is used to parse all compose files in all references, including old release branches and tags which will still have these files.

Comment (MaciejTe) [reply extra/release_tool.py]: got it, fixed

**Review by alfrunes — APPROVED** [review]

**Pull request merged**